SPECS/libyang/libyang.spec

```diff
@@ -0,0 +1,103 @@
+# valgrind finds invalid writes in libcmocka on arm and power
+# see bug #1699304 for more information
+%ifarch %{arm} ppc64le
+%global run_valgrind_tests OFF
+%else
+%global run_valgrind_tests ON
+%endif
+Summary:        YANG data modeling language library
+Name:           libyang
+Version:        2.0.231
+Release:        1%{?dist}
+License:        BSD-3-Clause
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/CESNET/libyang
+Source:         https://github.com/CESNET/libyang/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  cmake
+BuildRequires:  doxygen
+BuildRequires:  gcc
+BuildRequires:  libcmocka-devel
+BuildRequires:  make
+BuildRequires:  pkgconfig
+BuildRequires:  valgrind
+BuildRequires:  pkgconfig(libpcre2-8) >= 10.21
+
+%package devel
+Summary:        Development files for libyang
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Requires:       pcre2-devel
+
+%package devel-doc
+Summary:        Documentation of libyang API
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%package tools
+Summary:        YANG validator tools
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description devel
+Headers of libyang library.
+
+%description devel-doc
+Documentation of libyang API.
+
+%description tools
+YANG validator tools.
+
+%description
+Libyang is YANG data modeling language parser and toolkit
+written (and providing API) in C.
+
+%prep
+%autosetup -p1
+
+%build
+mkdir build
+pushd build
+%cmake \
+   -DCMAKE_INSTALL_PREFIX:PATH=%{_prefix} \
+   -DCMAKE_BUILD_TYPE=RELWITHDEBINFO \
+   -DENABLE_LYD_PRIV=ON \
+   -DENABLE_VALGRIND_TESTS=%{run_valgrind_tests} ..
+%make_build
+make doc
+
+%check
+ctest --output-on-failure -V %{?_smp_mflags}
+
+%install
+pushd build
+%make_install
+popd
+mkdir -m0755 -p %{buildroot}/%{_docdir}/libyang
+cp -a doc/html %{buildroot}/%{_docdir}/libyang/html
+
+%files
+%license LICENSE
+%{_libdir}/libyang.so.2
```

Comment:
If this file's a symbolic link, then it should probably go to the `devel` subpackage.

Reply:
Why? This will be needed at runtime for binaries that need libyang.so.2. (This is just standard SO versioning, right? I don't think there's anything unusual here)

Reply:
You're right, I got all mixed up here. Sorry for the trouble.